packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/action_validation/document_create_transition_action/structure_v0/mod.rs

```diff
@@ -78,35 +78,32 @@ impl DocumentCreateTransitionActionStructureValidationV0 for DocumentCreateTrans
         // Validate timestamps against block time
         // we do validation here but not in validate state because it's a cheap validation
         // and validate state implements expensive validation only
-        let latest_block_time_ms = platform.state.last_committed_block_time_ms();
+        let latest_block_time_ms = platform.state.any_block_info().time_ms;
         let average_block_spacing_ms = platform.config.block_spacing_ms;
 
         // We do not need to perform these checks on genesis
-        if let Some(latest_block_time_ms) = latest_block_time_ms {
-            let validation_result = check_created_inside_time_window(
-                self,
-                latest_block_time_ms,
-                average_block_spacing_ms,
-                platform_version,
-            )?;
-            if !validation_result.is_valid() {
-                return Ok(validation_result);
-            }
-
-            let validation_result = check_updated_inside_time_window(
-                self,
-                latest_block_time_ms,
-                average_block_spacing_ms,
-                platform_version,
-            )?;
-
-            if !validation_result.is_valid() {
-                return Ok(validation_result);
-            }
+        let validation_result = check_created_inside_time_window(
+            self,
+            latest_block_time_ms,
+            average_block_spacing_ms,
+            platform_version,
+        )?;
+        if !validation_result.is_valid() {
+            return Ok(validation_result);
```

Comment:
you changed the meaning of this code, please use last_committed_block_time_ms again.

Reply:
It's logically wrong to use last committed block info since it might not be any blocks yet. In this case you should use genesis information.

Reply:
Please do it in a separate PR.